client/dashboard/sites/index.tsx

```diff
@@ -73,10 +74,10 @@ const DEFAULT_FIELDS = [
 		filterBy: {
```

Comment:
For later, I would love to be able to drop the "render" function entirely here. In other words if the "type" is boolean, we define an automatic render function based on the `getValue`